.github/workflows/ci_be.yml

```diff
@@ -39,4 +39,4 @@ jobs:
 
       - name: Test with pytest
         run: |
-          pytest
+          pytest --ignore=tests/test_e2e.py
```

Comment:
So would you like creating a new pipeline for E2E? Will u do it this iteration? @Stanford997

Reply:
Yes. Im working on it.